gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -536,10 +536,10 @@ def _get_freq_by_pop(
     # Convert CMH result pandas DataFrame to a Table and restructure the annotation.
     cmh_ht = hl.Table.from_pandas(df)
     cmh_ht = cmh_ht.key_by("tmp_idx")
+    chisq_expr = cmh_ht[_ht.tmp_idx].statistic
+    chisq_expr = hl.or_missing(~hl.is_nan(chisq_expr), chisq_expr)
```

Comment:
why did we not previously set this to missing if the statistic was NaN?

Reply:
we were setting NaN to missing for the final here [https://github.com/broadinstitute/gnomad_qc/blob/817f991b7abd91978bb23ca0d2aaba1c8[…]b/gnomad_qc/v4/create_release/create_combined_faf_release_ht.py](https://github.com/broadinstitute/gnomad_qc/blob/817f991b7abd91978bb23ca0d2aaba1c8b4376eb/gnomad_qc/v4/create_release/create_combined_faf_release_ht.py#L646)